modules/phenyl-api-explorer/README.md

```diff
@@ -0,0 +1,32 @@
+# phenyl-api-explorer
+Automatically renders API explorer pages like [swagger UI](https://github.com/swagger-api/swagger-ui).
+
+![screenshot.png](https://github.com/phenyl-js/phenyl/raw/master/modules/phenyl-api-explorer/examples/screenshot.png)
```

Comment:
This image cannot load in this branch.
It can load when this branch merged to master.

In this branch, actual URL is: https://github.com/phenyl-js/phenyl/raw/add-readme-to-api-explorer/modules/phenyl-api-explorer/examples/screenshot.png

master -> add-readme-to-api-explorer